ts/DataGrid/DataGrid.ts

```diff
@@ -956,6 +980,89 @@ class DataGrid {
             this.headerContainer,
             this.outerContainer
         );
+
+        this.updateColumnHeaders();
+    }
+
+
+    /**
+     * Updates the column headers of the table.
+     * @internal
+     */
+    private updateColumnHeaders(): void {
```

Comment:
I'm not sure if this only updated the header. I would change the name and would try to split it into smaller methods responsible for a given element.

Reply:
BTW Do we have any plans for row headers?